static/js/modules/helpers.js

```diff
@@ -216,6 +215,41 @@ function zeroPad(container, item, appendee) {
   });
 }
 
+function amendmentVersion(data) {
```

Comment:
Could there be some variable name that'd be more expressive than `data`? It seems like a flag for whether the version is the current one. So maybe, `isCurrent`?